components/generic/InfoMessage.vue

```diff
@@ -23,4 +23,8 @@
   .alert.alert-light {
     font-style: italic;
   }
+
+  .alert-light {
+    color: black;
```

Comment:
You can add the color: black to the declaration above. 
And, let's use $black. That is slightly off black, but it is the "Europeana black".